swagger/v2/defendant.json

```diff
@@ -60,33 +60,48 @@
       ]
     },
     "maat_reference": {
-      "example": 314159265,
-      "description": "The LAA issued reference to the application. CDA expects a numeric number, although HMCTS allows strings",
-      "anyOf":[
-        {
-          "$ref": "laa_reference.json#/definitions/maat_reference"
-        },{
-          "type": "null"
-        }
-      ]
-    },
+     "example": 314159265,
```

Comment:
Has rubocop changed indentation to 1 space? This is no longer consistent with the rest of the file's indentation

Reply:
good spot!